From: =?utf-8?q?David_Pr=C3=A9vot?= <taffit@debian.org>
Date: Wed, 19 Mar 2014 11:07:08 -0400
Subject: Use ClassLoader from Symfony instead of autoload

Work around the lack of proper autoload.php from composer by using the
ClassLoader element from Symfony.

http://symfony.com/doc/current/components/class_loader/class_loader.html

Forwarded: not-needed
---
 tests/bootstrap.php | 22 +++++++++-------------
 1 file changed, 9 insertions(+), 13 deletions(-)

diff --git a/tests/bootstrap.php b/tests/bootstrap.php
index a01f8f9..46ff855 100644
--- a/tests/bootstrap.php
+++ b/tests/bootstrap.php
@@ -2,19 +2,15 @@
 
 date_default_timezone_set('UTC');
 
-$try = array(
-    __DIR__ . '/../vendor/autoload.php',
-    __DIR__ . '/../../../autoload.php',
-);
-
-foreach($try as $path) {
-    if (file_exists($path)) {
-        $autoLoader = include $path;
-        break;
-    }
-}
-
-$autoLoader->addPsr4('Sabre\\VObject\\',__DIR__ . '/VObject');
+require_once 'Symfony/Component/ClassLoader/ClassLoader.php';
+use Symfony\Component\ClassLoader\ClassLoader;
+$loader = new ClassLoader();
+$loader->setUseIncludePath(true);
+$loader->register();
+$loader->addPrefixes(array(
+// ClassLoader does not yet support PSR-4
+//	'Sabre' => __DIR__.'/../lib',
+));
 
 if (!defined('SABRE_TEMPDIR')) {
   define('SABRE_TEMPDIR', __DIR__ . '/temp/');
